xml/System/Uri.xml

```diff
@@ -188,6 +188,36 @@ Uri uri1 = new Uri("C:/test/path/file.txt") // Implicit file path.
 Uri uri2 = new Uri("file:///C:/test/path/file.txt") // Explicit file path.
 ```
  These implicit file paths are not compliant with the URI specification and so should be avoided when possible. When using .NET Core on Unix-based systems, implicit file paths can be especially problematic, because an absolute implicit file path is *indistinguishable* from a relative path. When such ambiguity is present, <xref:System.Uri> default to interpreting the path as an absolute URI.
+
+## Security Considerations
+
+Because of security concerns, your application should use caution when accepting <see cref="T:System.Uri" /> instances from untrusted sources and with <paramref name="dontEscape" /> set to <see langword="true" />.You can check a URI string for validity by calling the <see cref="M:System.Uri.IsWellFormedOriginalString" /> method.
```

Comment:
The `see crefs` aren't rendering, because this is markdown. Preview: https://review.docs.microsoft.com/en-us/dotnet/api/system.uri?view=netframework-4.8&branch=pr-en-us-4232#security-considerations

```suggestion
Because of security concerns, your application should use caution when accepting <xref:System.Uri> instances from untrusted sources and with `dontEscape` set to `true`. You can check a URI string for validity by calling the <xref:System.Uri.IsWellFormedOriginalString%2A> method.
```